tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -42,7 +42,7 @@ public function testDefault(): void
             ],
             'google_authenticator' => [
                 'enabled' => false,
-                'ip_white_list' => ['127.0.0.1'],
+                'trusted_ip_list' => ['127.0.0.1'],
```

Comment:
```suggestion
                'ip_white_list' => [],
                'trusted_ip_list' => ['127.0.0.1'],
```

Since `ip_white_list` is still accepted, it will be in the default values, with this the test should be fixed.

Reply:
Fixed